[bug] Error installing conan on Big Sur running on Apple Silicon

### Environment Details (include every applicable attribute)
  * Operating System+version: macOS Big Sur 11.0 Beta (20A5299w) on Apple A12Z Bionic
  * Compiler+version: Apple clang version 12.0.0 (clang-1200.0.22.19)
  * Conan version: 1.26.1
  * Python version: 3.8.2

### Steps to reproduce (Include if Applicable)
sudo pip3 install conan

### Logs (Executed commands with output) (Include/Attach if Applicable)
The problem is a compile error building "cryptography". I realize that this might be a Python issue with a conan requirement and not directly related to the conan project. But it is keeping me from building any of our projects that use conan on Apple Silicon systems.

If you feel this should reported to another project please let me know.

In the meantime, here's the build log (search below for 'error:' without quotes):

```
  Building wheel for cryptography (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /Applications/Xcode-beta.app/Contents/Developer/usr/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/tmp/pip-install-kkmw5tcb/cryptography/setup.py'"'"'; __file__='"'"'/private/tmp/pip-install-kkmw5tcb/cryptography/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /private/tmp/pip-wheel-jnxw961w --python-tag cp38
       cwd: /private/tmp/pip-install-kkmw5tcb/cryptography/
  Complete output (150 lines):
  c/_cffi_backend.c:6245:9: warning: 'ffi_prep_closure' is deprecated [-Wdeprecated-declarations]
      if (ffi_prep_closure(closure, &cif_descr->cif,
          ^
  /Applications/Xcode-beta.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/ffi/ffi.h:350:18: note: 'ffi_prep_closure' has been explicitly marked deprecated here
    __attribute__((deprecated))
                   ^
  1 warning generated.
  c/_cffi_backend.c:6245:9: error: implicit declaration of function 'ffi_prep_closure' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
      if (ffi_prep_closure(closure, &cif_descr->cif,
          ^
  1 error generated.
  Traceback (most recent call last):
    File "/Applications/Xcode-beta.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/distutils/unixccompiler.py", line 117, in _compile
      self.spawn(compiler_so + cc_args + [src, '-o', obj] +
    File "/Applications/Xcode-beta.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/distutils/ccompiler.py", line 910, in spawn
      spawn(cmd, dry_run=self.dry_run)
    File "/Applications/Xcode-beta.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/distutils/spawn.py", line 36, in spawn
      _spawn_posix(cmd, search_path, dry_run=dry_run)
    File "/Applications/Xcode-beta.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/distutils/spawn.py", line 157, in _spawn_posix
      raise DistutilsExecError(
  distutils.errors.DistutilsExecError: command 'xcrun' failed with exit status 1
```

--- a/conans/requirements.txt
+++ b/conans/requirements.txt
@@ -14,7 +14,4 @@ deprecation>=2.0, <2.1
 tqdm>=4.28.1, <5
 Jinja2>=2.9, <3
 python-dateutil>=2.7.0, <3
-idna==2.6 ; sys_platform == "darwin" # Solving conflict, somehow is installing 2.7 when requests require 2.6
-cryptography>=3.2, <4 ; sys_platform == "darwin"
-pyOpenSSL>=16.0.0, <19.0.0 ; sys_platform == "darwin"
 

